csharp/ql/lib/semmle/code/csharp/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -410,6 +410,27 @@ module LocalFlow {
     n instanceof SummaryNode or
     n instanceof ImplicitCapturedArgumentNode
   }
+
+  /**
+   * Gets a node that may execute last in `n`, and which, when it executes last,
+   * will be the value of `n`.
+   */
+  private ControlFlow::Nodes::ExprNode getALastEvalNode(ControlFlow::Nodes::ExprNode cfn) {
+    any(LocalExprStepConfiguration x).hasExprPath(_, result, any(ConditionalExpr ce), cfn)
```

Comment:
We should also include
- `NullCoalescingExpr::getAnOperand`
- `SwitchExpr::getACase`
- `SwitchCaseExpr::getBody`

(And ideally also tests.)

Reply:
Alright - I will look into it!